src/components/Footer.js

```diff
@@ -12,34 +17,35 @@ const Footer = ({ currentPin, displayInformation, handleClick, marker }) => {
   );
 
   const defaultFooter = (
-    <React.Fragment>
+    <section>
+    <div>
```

Comment:
What's the reason for having a `<div>` nested inside the `<section>`? I can imagine wanting that if you want the content inside the `<div>` to scroll inside the `<section>` container, but that's not currently happening.